FlowCrypt/Controllers/Settings/KeySettings/Key Details/Nodes/KeyTextCellNode.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  KeyTextCellNode.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 12/13/19.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import AsyncDisplayKit
+
+final class KeyTextCellNode: CellNode {
```

Comment:
We should probably start putting similar classes into a folder of generic reusable nodes.

Because it would not make sense to have a KeyTextCellNode, then AnotherTextCellNode, then MenuListItemCellNode etc if they all did the same things.

Reply:
We already have a folder for Nodes that we use in different places. Some common nodes
Common UI -> Nodes

But I want to keep specific UI elements that stick to this class in a folder where it belongs only to one view controller. 
If the node can be reused I usually put it to Common UI folder